kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -53,13 +60,13 @@ define([
                     auto: true,
                 },
                 {
-                    mode: 'complete',
+                    mode: 'completed',
```

Comment:
I changed the FSM state names to match the job state names -- having different names was a needless complication.